.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### ISSUE TITLE
+
+ ## Checklist
+- [ ] I have read this repository's contribution guidelines.
+- [ ] My pull request has a descriptive title (not a vague title like Update index.MD)
+- [ ] My pull request targets the dev branch of this [project](https://github.com/didierganthier/cook-snap).
+- [ ] I added relevant documentation for all the files I changed
+
+## Summary
+This PR adds several files and folders to set up the architecture of the app
+
+ ## What changes have been made?
+ Add relevant changes
+- [example](point.to.url)
+
+## Author
+- Your
+
+## Reviewer
+- @didierganthier
```

Comment:
Well done adding the issue template 👍🏽 it helps understand better what to review